# src/services/standard/feedparser.cpp:52:38: error: format not a string literal and no format arguments [-Werror=format-security]
Index: rssguard-3.4.1/src/services/standard/feedparser.cpp
===================================================================
--- rssguard-3.4.1.orig/src/services/standard/feedparser.cpp	2017-07-13 06:52:41.000000000 +0200
+++ rssguard-3.4.1/src/services/standard/feedparser.cpp	2017-07-15 16:41:04.607997177 +0200
@@ -49,7 +49,7 @@
       messages.append(new_message);
     }
     catch (const ApplicationException &ex) {
-      qDebug(qPrintable(ex.message()));
+      qDebug("%s", qPrintable(ex.message()));
     }
   }
 
